sources/fr/countrywide.json

```diff
@@ -0,0 +1,33 @@
+{
+  "coverage": {
+      "ISO 3166": {
+          "alpha2": "FR-01"
+      },
+      "country": "fr"
+  },
+  "website": "https://adresse.data.gouv.fr",
+  "data": "https://adresse.data.gouv.fr/data/ban/adresses/latest/csv/adresses-france.csv",
+  "protocol": "http",
+  "conform": {
+      "format": "csv",
+      "csvsplit": ";",
+      "id": "id",
+      "number": {
+        "function": "join",
+        "fields": ["numero", "rep"],
+        "separator": ""
+      },
+      "street": "nom_voie",
+      "lon": "lon",
+      "lat": "lat",
+      "city": "nom_commune",
+      "postcode": "code_postal"
+  },
+  "email": "adresse@data.gouv.fr",
```

Comment:
Per https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md, The email field says `Do not set unless the data provider wants to receive updates.` his the data provider opted in to this? Otherwise if it's just a generic contact email, then best to put this in the `contact` key.

Reply:
We can accept this kind of notification in the future.
But since it's not implemented for now, I just moved to the contact email.